jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -798,12 +798,12 @@ properties:
             fits_keyword: GROUPGAP
             blend_table: True
           drop_frames1:
-            title: Frames dropped at start of each integration
+            title: Number of frames dropped prior to first integration
             type: integer
             fits_keyword: DRPFRMS1
             blend_table: True
           drop_frames3:
-            title: Frames dropped at end of each integration
+            title: Number of frames dropped between integrations
```

Comment:
For consistency with the above, let's make this "Frames dropped between integrations"